extensions/iceberg/s3/src/test/java/io/deephaven/iceberg/util/IcebergToolsTest.java

```diff
@@ -715,4 +715,143 @@ public void testOpenAllTypesTable() throws ExecutionException, InterruptedExcept
         final io.deephaven.engine.table.Table table = adapter.readTable(tableId, instructions);
         Assert.eq(table.size(), "table.size()", 10, "10 rows in the table");
     }
+
+    @Test
+    public void testTableDefinition() throws ExecutionException, InterruptedException, TimeoutException {
+        uploadParquetFiles(new File(IcebergToolsTest.class.getResource("/warehouse/sales/sales_multi").getPath()),
+                warehousePath);
```

Comment:
Do we need to upload parquet files to get the definition? I think it's all involved in the metadata (which we aren't uploading as part of this).